src/Gateways/PorticoConnector.ts

```diff
@@ -417,6 +417,11 @@ export class PorticoConnector extends XmlGateway implements IPaymentGateway {
       subElement(cardData, "TokenRequest").append(
         cData(builder.requestMultiUseToken ? "Y" : "N"),
       );
+
+      if(builder.tokenUqinue){
+        subElement(subElement(cardData, "TokenParameters"),
```

Comment:
Suggest it's need to extract the separate elements.

Reply:
Resolved